source/server/backtrace.h

```diff
@@ -86,12 +88,15 @@ class BackwardsTrace : Logger::Loggable<Logger::Id::backtrace> {
     // work when installed.
     ENVOY_LOG(critical, "Backtrace obj<{}> thr<{}>:", obj_name, thread_id);
 #else
-    ENVOY_LOG(critical, "Backtrace obj<{}> thr<{}> (use tools/stack_decode.py):", obj_name,
+    char out[200];
+    ENVOY_LOG(critical,
+              "Backtrace obj<{}> thr<{}> (If unsymbolized, use tools/stack_decode.py):", obj_name,
```

Comment:
While you're here, should we perhaps flip the order from `"Backtrace obj<{}> thr<{}> ..."` to `"Backtrace thr<{}> obj<{}> ..."` to match rest of the output?